scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml

```diff
@@ -63,6 +64,11 @@
               export RPM_DEPCHECK_MEDIADEPS=" "
               export SRCRPM_DEPCHECK_MEDIADEPS=" "
               export INSTALLCHECK_EXTRA_OPTS="--exclude pattern:"
+          elif [[ "$project" =~ Devel:Cloud:(.*):Appliances ]]; then
+              cloud_version=${BASH_REMATCH[1]}
+              export RPM_DEPCHECK_MEDIADEPS="solv/cloud-$cloud_version-dvd.solv solv/cloud-$cloud_version-updates.solv"
```

Comment:
I have the SES packages in there, but without the packages that are also in Cloud -- so these packages actually depend on Cloud packages.

But I'm thinking of just putting all packages we need (including Cloud ones) in the deps iso and only use this as the one true source of packages that replaces everything for the appliance.